src/k8s-engine/kim.ts

```diff
@@ -368,23 +394,28 @@ class Kim extends EventEmitter {
     args.push(taggedImageName);
     args.push(dirPart);
 
-    return await this.runCommand(args);
+    return await this.runKimCommand(args);
   }
 
   async deleteImage(imageID: string): Promise<childResultType> {
-    return await this.runCommand(['rmi', imageID]);
+    return await this.runKimCommand(['rmi', imageID]);
   }
 
   async pullImage(taggedImageName: string): Promise<childResultType> {
-    return await this.runCommand(['pull', taggedImageName, '--debug']);
+    return await this.runKimCommand(['pull', taggedImageName, '--debug']);
   }
 
   async pushImage(taggedImageName: string): Promise<childResultType> {
-    return await this.runCommand(['push', taggedImageName, '--debug']);
+    return await this.runKimCommand(['push', taggedImageName, '--debug']);
   }
 
   async getImages(): Promise<childResultType> {
-    return await this.runCommand(['images', '--all'], false);
+    return await this.runKimCommand(['images', '--all'], false);
+  }
+
+  async scanImage(taggedImageName: string): Promise<childResultType> {
+    return await this.runTrivyCommand(['image', '--no-progress', '--format', 'template',
+      '--template', '@/var/lib/trivy.tpl', taggedImageName]);
```

Comment:
Hmm, this means we can have the template be a JavaScript string that gets passed in, right? (I'm mainly thinking of using something like `file-loader`, so we can copy one less file into the VM.)